bigquery/tables.js

```diff
@@ -184,18 +184,40 @@ function exportTableToGCS (options, callback) {
   });
 }
 // [END export_table_to_gcs]
+
+// [START insert_rows_as_stream]
+/**
+ * Insert rows (as a stream) into a BigQuery table.
+ * @param {object} options Configuration options.
+ * @param {array} options.rows An array of rows to insert into a BigQuery table.
+ * @param {string} options.dataset The ID of the dataset containing the target table.
+ * @param {string} options.table The ID of the table to insert rows into.
+ * @param {function} callback Callback function to receive query status.
+ */
+function insertRowsAsStream (options, callback) {
+  var table = bigquery.dataset(options.dataset).table(options.table);
+  table.insert(options.rows, function (err, insertErrors) {
+    if (err) {
+      return callback(err);
+    }
+    return callback(null, insertErrors);
```

Comment:
We should have a `console.log` above this line like all the other samples, e.g.:

```js
if (err) {
  return callback(err);
}

console.log('Inserted %d row(s)!', options.rows.length);
return callback(null, insertErrors);
```